tests/core/Jalasoft.TeamUp.Projects.Core.Tests/Validators/ProjectValidatorTests.cs

```diff
@@ -0,0 +1,159 @@
+        public void Should_Have_Error_When_Name_is_Null()
+        {
+            // Arrange
+            var project = new Project { Name = null };
+
+            // Act
+            var result = this.validator.TestValidate(project);
+
+            // Assert
+            result.ShouldHaveValidationErrorFor("Name");
+        }
+
+        [Fact]
+        public void Should_Have_Error_When_Name_Contain_Numbers()
+        {
+            // Arrange
+            var project = new Project { Name = "TeamUp 78484" };
+
+            // Act
+            var result = this.validator.TestValidate(project);
+
+            // Assert
+            result.ShouldHaveValidationErrorFor("Name");
+        }
+
+        [Fact]
+        public void Should_Have_Error_When_Description_is_More_Than_160_Characters()
+        {
+            // Arrange
+            var project = new Project { Description = "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam faucibus in eros eu hendrerit. Nulla a nunc eget est tempor placerat at nec diam. Donec at tincidunt." };
+
+            // Act
+            var result = this.validator.TestValidate(project);
+
+            // Assert
+            result.ShouldHaveValidationErrorFor(project => project.Description);
+            result.ShouldHaveValidationErrorFor(project => project.Description).WithErrorMessage("It must not have more than 160 characters");
+        }
+
+        [Fact]
+        public void Should_Have_Error_When_Text_Invitation_is_More_Than_160_Characters()
+        {
+            // Arrange
+            var project = new Project { TextInvitation = "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam faucibus in eros eu hendrerit. Nulla a nunc eget est tempor placerat at nec diam. Donec at tincidunt." };
+
+            // Act
+            var result = this.validator.TestValidate(project);
+
+            // Assert
+            result.ShouldHaveValidationErrorFor(project => project.TextInvitation);
+            result.ShouldHaveValidationErrorFor(project => project.TextInvitation).WithErrorMessage("It must not have more than 160 characters");
+        }
+
+        [Fact]
+        public void Should_Have_Error_When_Extesion_logo_is_Invalid()
+        {
+            // Arrange
+            var project = new Project { Logo = "noextensionvalid.com" };
+
+            // Act
+            var result = this.validator.TestValidate(project);
+
+            // Assert
+            result.ShouldHaveValidationErrorFor("Logo");
+        }
+
+        [Fact]
+        public void Should_Have_Error_When_Date_is_Invalid()
+        {
+            // Arrange
+            var project = new Project { CreationDate = default(DateTime) };
+
+            // Act
+            var result = this.validator.TestValidate(project);
+
+            // Assert
+            result.ShouldHaveValidationErrorFor(project => project.CreationDate);
+        }
+
+        [Fact]
+        public void Should_Not_Have_Pass_When_State_is_Bool()
```

Comment:
please change the title, it mentions that should not have to pass but the assert does the opposite

Reply:
The name was changed